tests/test_1399_from_jax.py

```diff
@@ -1,49 +0,0 @@
-# BSD 3-Clause License; see https://github.com/scikit-hep/awkward/blob/main/LICENSE
```

Comment:
this should be kept as well if we keep `ak.from_jax`

Reply:
agree, done